ui/packages/lib/src/components/Login.js

```diff
@@ -40,12 +39,7 @@ export const Login = ({ location }) => {
           />
           <EuiFlexGroup direction="column" alignItems="center">
             <EuiFlexItem grow={false} style={{ maxWidth: "200px" }}>
-              <GoogleLogin
-                clientId={clientId}
-                buttonText="Login with Google"
-                onSuccess={onLogin}
-                onFailure={onFailure}
-              />
+              <GoogleLogin onSuccess={onLogin} onError={onFailure} />
```

Comment:
Should we consider setting `isSignedIn={true}` too? My understanding is that setting this property will also be able to look at other apps that the user is signed into (such as Gmail) and skip the login prompt if they have already authorised our app once.

Reply:
Not sure what you mean by this. There is no `isSignedIn` property on the `GoogleLogin` component: https://www.npmjs.com/package/@react-oauth/google

Reply:
Oops I opened the wrong link from the PR description: https://www.npmjs.com/package/react-google-login

I think it's called `auto_select` in the new one: https://www.npmjs.com/package/@react-oauth/google

Reply:
`auto_select` only works with `useOneTap` enabled. I'm not sure if I like this behavior. 

Yes, it saves two clicks in the sign-in process, but it also slows down the sign-in, because a user has to wait a few seconds before signing in. I'd rather prefer to keep the process as is.

![Aug-08-2022 10-19-24](https://user-images.githubusercontent.com/1886194/183325489-549cbbc5-578d-45b7-a79e-464373bc3673.gif)

Reply:
Ok, I agree with leaving the behaviour as it is.

Reply:
Although perhaps @deadlycoconuts has some questions? :)

Reply:
Ahah, nah I'm fine either way with or without the auto sign-in. I just wasn't expecting the auto sign-in to be so slow :/

Reply:
>  I just wasn't expecting the auto sign-in to be so slow :/

Perhaps, this was done intentionally to give time to users to cancel the auto-sign-in. But then, it fails the purpose